File: test_db.mjs
import { createClient } from '@supabase/supabase-js';
import dotenv from 'dotenv';
dotenv.config();

const supabase = createClient(process.env.SUPABASE_URL, process.env.SUPABASE_SERVICE_ROLE_KEY);

async function run() {
  const { data: player } = await supabase.from('players').select('id, team_id, short_name').ilike('short_name', '%Pino%').limit(1).single();
  console.log("Player:", player);
  
  const { data: scores } = await supabase.from('player_scores').select('fixture_id').eq('player_id', player.id);
  console.log("Scores:", scores);
  
  const { data: fixtures } = await supabase.from('fixtures').select('id, matchday, status').eq('status', 'finished').or(`home_team_id.eq.${player.team_id},away_team_id.eq.${player.team_id}`);
  console.log("Fixtures:", fixtures);
}
run();

File: test_db.py
import os
from supabase import create_client
from dotenv import load_dotenv

load_dotenv()
sb = create_client(os.environ["SUPABASE_URL"], os.environ["SUPABASE_SERVICE_ROLE_KEY"])

p = sb.table("players").select("id, team_id, short_name").ilike("short_name", "%Pino%").limit(1).execute().data[0]
print("Player:", p)

scores = sb.table("player_scores").select("fixture_id").eq("player_id", p["id"]).execute().data
print("Scores:", scores)

fixtures = sb.table("fixtures").select("id, matchday, status, home_team_id, away_team_id").eq("status", "finished").or_(f"home_team_id.eq.{p['team_id']},away_team_id.eq.{p['team_id']}").execute().data
print("Fixtures:", fixtures)
